karma.conf.js

```diff
@@ -88,6 +90,21 @@ module.exports = function(config) {
          * If you have a different webpack.config.js file that's used for testing
          * purposes, you can specify that here.
          */
-        webpack: webpackConfig.test
+        webpack: webpackConfig.test,
+
+        // How long will Karma wait for a message from a browser before disconnecting from it (in ms).
+        browserNoActivityTimeout: 180000,
+        captureTimeout: 180000,
+        retryLimit: 5,
+        customLaunchers: {
+            Chrome_headless: {
+                base: 'Chrome',
+                flags: [
+                    '--headless',
+                    '--disable-gpu',
+                    ' --remote-debugging-port=9222'
```

Comment:
Not sure if these are still needed in Chrome 60. Won't hurt anyways.

Reply:
Actually, I have used the [following](https://github.com/telerik/kendo-angular-tasks/blob/develop/karma.conf.js) as a reference point :smile_cat: